python/tk_multi_workfiles/work_files.py

```diff
@@ -17,70 +17,6 @@
 from .util import report_non_destroyed_qobjects
 
 
-class TimedGc(QtCore.QObject):
```

Comment:
All this timed garbaged collection code doesn't work with non-modal dialog and forcing garbage collection doesn't help reclaimed leaked models unfortunately.